server/raft.go

```diff
@@ -819,7 +859,7 @@ func (n *raft) PauseApply() error {
 		n.stepdown.push(noLeader)
 	}
 
-	n.debug("Pausing our apply channel")
+	n.debug("Pausing our apply queue")
```

Comment:
These are tough, sometimes easy to see a better log line but if its out there folks could depend on it, so I usually shy away from changing.

Reply:
Good point, I've put them back to what they were.